src/main/kotlin/global/InstallationManager.kt

```diff
@@ -62,7 +62,7 @@ object InstallationManager {
 
     /** Gets the version of the wurstscript.jar via cli */
     fun getVersionFomJar() {
-        val proc = Runtime.getRuntime().exec("java -jar " + compilerJar.toAbsolutePath() + " --version")
+        val proc = Runtime.getRuntime().exec("java -jar \"" + compilerJar.toAbsolutePath() + "\" --version")
```

Comment:
Probably better to pass an array and let Java do it automatically:

    val proc = Runtime.getRuntime().exec(arrayOf("java", "-jar", InstallationManager.compilerJar.toAbsolutePath().toString(), "--version"))